backend/tests/services/execution/test_request_builder_skill_identity_protocol.py

```
@@ -0,0 +1,23 @@
+    request = ExecutionRequest(
+        task_id=1,
+        subtask_id=2,
+        user_id=3,
+        user_name="alice",
+        skill_identity_token="skill-jwt",  # noqa: S106
+    )
+
+    data = request.to_dict()
+    restored = ExecutionRequest.from_dict(data)
+
+    assert restored.skill_identity_token == "skill-jwt"
```

Ruff (0.15.7)

[error] Possible hardcoded password assigned to: "skill_identity_token"

(S105)